File: EJERCICIOS DE LOS ALUMNOS/JON/Python/03 Condicionales IF ELIF ELSE/Notas.py
print("Ejercicio Notas")
print("")
print("")

nota=float(input("introduce tu nota: "))

if nota>=4.5:
    print("Estás aprobado.")
    
else:
    print("Estás suspenso.")
    
if 10>nota<0:
    print("Nota no válida")
else:
    if 0>nota>4.4:
        print("Suspenso.")    
    if 4.5<nota<5.5:
        print("Aprobado.")
    if 5.5<nota<6.5:
        print("Bien.")
    if 6.6<nota<8.4:
        print("Notable.")
    if 8.5<nota<10:
        print("Sobresaliente.")

File: EJERCICIOS DE LOS ALUMNOS/RAFA/JAVA/ProgramacionSistemas/src/ejerciciosObligatoriosMetodos/Notas.java
package ejerciciosObligatoriosMetodos;

public class Notas {
	public int nota(int a) {
		if (a<=4){
			System.out.println(" estas suspenso");
		}else if (a>=4 && a<=5) {
			System.out.println(" suficiente");
		}else if (a==6) {
            System.out.println(" bien");
		}else if (a>=7 && a<=8) {
            System.out.println(" notable");
		}else if (a>=9 && a<=10) {
            System.out.println(" notable");
		}else {
            System.out.println("prueba del 0 al 10");
        }
	return a;
	}

}